graphs/possible_bipartition.py

```diff
@@ -1,12 +1,56 @@
 # Can be used for BFS
 from collections import deque 
+class Graph():
 
-def possible_bipartition(dislikes):
+    def __init__(self, V):
+            self.V = V
+            self.graph = [[0 for column in range(V)] \
+                                    for row in range(V)]
```

Comment:
🤔 It looks like you're creating a graph here as an adjacency matrix with no edges initially.